scripts/example-mc.jl

```diff
@@ -0,0 +1,77 @@
+# Define parameters for MCMC simulation
+L = square_supercell_dimensions[1]
+M = square_supercell_dimensions[2]
+Ns = [4]
+adsorption_energy = -0.04
+nn_energy = -0.01
+nnn_energy = -0.0025
```

Comment:
There's a logic to these energies that should be reflected in the tutorial for MCMC in the documentation.